src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.User;
+
+public class ShoppingCartResponseDto {
+    private java.lang.Long id;
+    private List<Ticket> ticketIds;
+    private User user;
```

Comment:
Replace User with Long userId to store only user id instead of the whole User object.